src/public/modules/forms/admin/views/edit-start-page.client.modal.html

```diff
@@ -18,7 +18,7 @@ <h2 class="modal-title">Edit Welcome</h2>
               <label class="col-xs-12">
                 <input
                   type="radio"
-                  ng-change="vm.updateLogoUrl()"
+                  ng-change="vm.updateLogoUrl(vm.myform.startPage.logo.state)"
```

Comment:
since this happens when the user changes the radio, when they switches back to custom logo without saving, the uploaded file details will be cleared right?

i think the better place to clear the extra states should be just right before the endpoint is called

Reply:
good catch, will fix!